SEO fixes for docs

PrajwalPrashanth opened the pull request.

sidujjain commented on docs/api-reference/commit.md:
````diff
@@ -1,5 +1,7 @@
 
 ```eval_rst
+.. meta::
+   :description: Use Jovian commit to upload jupyter notebooks online
````
```suggestion
   :description: Use Jovian commit to upload and share Jupyter notebooks on Jovian platform.
```

sidujjain commented on docs/api-reference/oembed.md:
```diff
@@ -17,10 +17,10 @@ Parameters:
+  - [Jovian Notebook (Version specified)](https://jovian.ml/aakashns/movielens-fastai/v/14)
 - Jovian Viewer URLs
-  - [http://jovian.ml/viewer?url={notebook-url}](http://jovian.ml/viewer?url=https%3A%2F%2Fjvn.storage.googleapis.com%2Fgists%2Faakashns%2F5bc23520933b4cc187cfe18e5dd7e2ed%2Fraw%2F901a9d2508bd441dbf06954c5f46bf58%2Fmovielens-fastai.ipynb)
+  - [https://jovian.ml/viewer?url={notebook-url}](https://jovian.ml/viewer?url=https%3A%2F%2Fjvn.storage.googleapis.com%2Fgists%2Faakashns%2F5bc23520933b4cc187cfe18e5dd7e2ed%2Fraw%2F901a9d2508bd441dbf06954c5f46bf58%2Fmovielens-fastai.ipynb)
```
add text in []

PrajwalPrashanth replied:
That label for the link is a example of the URL format to be used, think it can be retained as above. Let me know if it needs to be changed.

sidujjain commented on docs/api-reference/oembed.md:
````diff
@@ -39,3 +39,8 @@ Response:
+
+```eval_rst
+.. meta::
+   :description: Embed online jupyter notebooks to blogs and websites
````
```suggestion
   :description: Embed Jupyter notebooks on blogs and websites.
```

sidujjain commented on docs/callbacks/fastai.md:
````diff
@@ -1,5 +1,7 @@
 
 ```eval_rst
+.. meta::
+   :description: Track hyperparameters, metrics with fastai callback by Jovian
````
```suggestion
   :description: Track hyper-parameters and metrics with Fastai callback by Jovian.
```

sidujjain commented on docs/callbacks/keras.md:
````diff
@@ -1,5 +1,7 @@
 
 ```eval_rst
+.. meta::
+   :description: Track hyperparameters, metrics with keras callback by Jovian
````
```suggestion
   :description: Track hyper-parameters and metrics with Keras callback by Jovian.
```

sidujjain commented on docs/integrations/conda.md:
````diff
@@ -1,3 +1,8 @@
+
+```eval_rst
+.. meta::
+   :description: Jovian.ml integration with anaconda.org
````
Jovian integration with Anaconda. 

Let's use "Jovian" (instead of Jovian.ml) and punctuation at the end for consistency.

sidujjain commented on docs/user-guide/attach.md:
````diff
@@ -1,5 +1,10 @@
 
+```eval_rst
+.. meta::
+   :description: Attach files, images, utils, python to online Jovian jupyter notebooks
````
```suggestion
   :description: Attach files, images and model outputs to Jovian projects.
```

sidujjain commented on docs/api-reference/logger.md:
````diff
@@ -1,6 +1,8 @@
 
 ```eval_rst
+.. meta::
+   :description: Use Jovian to track hyperparameters, metrics and dataset.
````
hyper-parameters
(change in other places too)

PrajwalPrashanth replied:
I think the general usage is `hyperparameters` including wikipedia. Would it be good for SEO, if we change it ?

sidujjain replied:
right. let's use hyperparameters then.

PrajwalPrashanth commented:
**Update**
- Added missing period punctuation
- Changed instances of `online jupyter notebooks` --> `Jupyter notebooks` as per the latest data it has more traffic and low competition too.
- Changed instances of `Jovian.ml` --> `Jovian`

sidujjain commented on docs/user-guide/track.md:
````diff
@@ -1,5 +1,10 @@
 
+```eval_rst
+.. meta::
+   :description: Track Datasets, Hyperparameters and Metrics for Jupyter notebooks.
````
```suggestion
   :description: Track datasets, hyperparameters and metrics for ML experiments.
```

sidujjain commented on docs/user-guide/compare.md:
````diff
@@ -1,5 +1,10 @@
 
+```eval_rst
+.. meta::
+   :description: Compare and analyze different versions of Jupyter notebooks with Jovian.
````
```suggestion
   :description: Compare and analyze different versions of ML experiments with Jovian.
```

sidujjain commented on docs/user-guide/collaborate.md:
````diff
@@ -1,5 +1,10 @@
 
+```eval_rst
+.. meta::
+   :description: Collaborate on Jupyter notebooks with Jovian platform.
````
```suggestion
   :description: Collaborate on Jupyter notebooks using Jovian platform.
```
